test/tracing/reactnativetracing.test.ts

```diff
@@ -117,6 +119,98 @@ describe('ReactNativeTracing', () => {
     jest.clearAllMocks();
   });
 
+  describe('trace propagation targets', () => {
```

Comment:
l/recommendation: Afaik, in the JS SDK we have tests that show which option is taken if multiple of these properties are specified. Admittedly it's an edge case, so feel free to disregard (not a blocker).